monkey/monkey_island/cc/repository/mongo_credentials_repository.py

```diff
@@ -0,0 +1,85 @@
+from typing import Sequence
+
+from pymongo import MongoClient
```

Comment:
`pymongo` isn't an explicit dependency of monkey island. I've added it to the `Pipfile`.